utf8/valid_fuzz_test.go

```diff
@@ -0,0 +1,25 @@
+//go:build go1.18
```

Comment:
Maybe call this file valid_go18_test.go, if I'm reading this right it's only separated because it requires go 1.18?

Reply:
Done. Wasn't sure where i was going with fuzzing.